todoman/cli.py

```diff
@@ -171,10 +171,12 @@ def cli(ctx, color, porcelain):
 @click.argument('summary', nargs=-1)
 @click.option('--list', '-l', callback=_validate_list_param,
               help='The list to create the task in.')
+@click.option('--location',
+              help='The location of the todo.')
```

Comment:
`todo edit` and `todo new` have a few options in common. We moved those options to a helper function here: https://github.com/pimutils/todoman/blob/master/todoman/cli.py#L82

All you need to do is add the option there, and also add `'location'` to the tuple at line 94.